src/components/EventModal.vue

```diff
@@ -185,6 +213,30 @@ export default {
 				if (this.description) {
 					event.addProperty(new TextProperty('DESCRIPTION', this.description))
 				}
+
+				if (this.attendeesList && this.attendeesList.trim() !== '') {
+					const organizerEmail = this.organizerEmail?.toLowerCase() || ''
+
+					const attendeesArray = this.attendeesList
+						.split(',')
+						.map(s => s.trim())
+						.filter(Boolean)
+
+					const filteredAttendees = attendeesArray.filter(att => att.toLowerCase() !== organizerEmail)
+
+					filteredAttendees.forEach(att => {
+						const value = att.includes('@') ? `mailto:${att}` : att
+						event.addProperty(new AttendeeProperty('ATTENDEE', value))
```

Comment:
```suggestion
						event.addProperty(new AttendeeProperty('ATTENDEE', `mailto:${att}`))
```

We don't need to catch that case. The URI should always start with a `mailto:`.

Reply:
Done